pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -81,9 +85,10 @@ export function readDeploymentProfiles(): settings.DeploymentProfileType {
     }
     break;
   case 'linux':
-    for (const rootPath of [paths.deploymentProfileSystem, paths.deploymentProfileUser]) {
-      profiles = readProfileFiles(rootPath, 'defaults.json', 'locked.json', JSON);
+    for (const configDir in linuxPaths) {
+      const basenames = linuxPaths[configDir];
 
+      profiles = readProfileFiles(configDir, basenames[0], basenames[1], JSON);
```

Comment:
I find this easier to understand:

```suggestion
      const [defaults, locked] = linuxPaths[configDir];

      profiles = readProfileFiles(configDir, defaults, locked, JSON);
```